### PR TITLE
chore: Upgrade to gs 9.54.0

### PR DESCRIPTION
**Pull request by mourad-tgl**
Upgrade to gs 9.54.0

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env bash
 # bin/compile <build-dir> <cache-dir>
 
-echo "-----> Installing Ghostscript 9.53.3"
+echo "-----> Installing Ghostscript 9.54.0"
 
 BUILD_DIR=$1
-PACKAGE="https://s3.amazonaws.com/tgl.public/buildpacks/ghostscript/ghostscript-9.53.3-linux-x86_64.tgz"
```
Comment (guillaume-tgl) [inline bin/compile:7]: @bvirlet do you remember why we kept the GS binary on S3? Just in case Artifex server is down at some point?

**Review by guillaume-tgl — APPROVED** [review]

**Comment by guillaume-tgl** [comment]
@mourad-tgl you should adapt the Readme as well

File: bin/compile (hunk 1 above)
Comment (mourad-tgl) [reply bin/compile:7]: Artifex is the maintainer of Ghostscript. It is unlikely to be down. However, I can move the release to our S3 if you grant me write access. :-)

Comment (guillaume-tgl) [reply bin/compile:7]: Let's leave it like this for now.

**Pull request merged**